src/mmap_unix.rs

```diff
@@ -510,6 +601,13 @@ mod tests {
         assert_eq!(r.prot(), libc::PROT_READ | libc::PROT_WRITE);
         assert_eq!(r.flags(), libc::MAP_NORESERVE | libc::MAP_PRIVATE);
         assert!(r.owned());
+
+        let builder = MmapRegionBuilder::new(0x1_0000, ())
```

Comment:
Can we test the builder with an actual bitmap as well?

Reply:
Hmm it looks like building with an actual `Bitmap` wouldn't make that much of a difference here because we're not testing the actual tracking functionality as well; the bitmap is just moved around. Did you have something more than that in mind?

Reply:
Ok we chatted a bit, and now I'm building and configuring the region with a non-trivial bitmap, calling `build`, and applying the generic volatile memory dirty tracking test on the result. I've also changed the default flags value for builder to `ANONYMOUS | PRIVATE` since that seems more like an  expected default value.